docs/src/faqs/migrate_from_fnapi.rst

```diff
@@ -233,85 +80,60 @@ The following is a collection of suggested changes to optimize Docker builds by
 
 .. caution::
 
-    **Most of these changes are incompatible with the FnAPI.**
-
-    The following changes will break your job if you return to using the FnAPI. If you choose to switch back to the FnAPI, simply undo these deletions.
-
-* **DELETE** any lines updating & installing Debian packages, i.e. ``apt-get update && apt-get install ...``. These commands have been moved to ``setup.py`` and will run via the added line ``RUN pip install .`` .
```

Comment:
We still need to prescribe them to add  `apt-get update and apt-get install` for any system-level deps (a required change). Otherwise they won't have them by the time they install their python dependencies.

An optional/suggested change that we could provide: add `apt-get update` (but not `apt-get install`) if they do not have any system-level dependencies to install, to keep their image updated.